### ISSUE TITLE
Notice when saving a model introduced in 3.0.4

### ISSUE THREAD

**Issue by terax6669** [phalcon/cphalcon#12688]
### Expected and Actual Behavior

Happens when trying to save empty string to a model.

```php
Undefined index: column_name in phalcon/mvc/model.zep on line 2092

```
> Provide minimal script to reproduce the issue

```php
class a extends \Phalcon\Mvc\Model {

    public $column_name;
}

$a = new $a;
$a->column_name = '';
$a->save();
```
### Details

* Phalcon version: 3.0.4 Powered by Zephir => Version 0.9.6a-dev-aef205594b
* PHP Version: PHP 7.0.16-3
* Operating System: Debian 8.7
* Installation type: https://packagecloud.io/phalcon/stable (3.0.4-2+extra)
* Server: Nginx
* Other related info (Database, table schema): 
PostgreSQL 9.6.0
Column definition: `column_name CHARACTER(3)`

**Comment by Jurigag** [comment]
Yea i checked it and it's happening, will try to fix.

**Comment by sergeyklay** [comment]
Fixed in the `3.1.x` branch.

**Comment by zGaron** [comment]
oops..
